packages/stream_chat/lib/src/client/client.dart

```diff
@@ -220,7 +227,7 @@ class StreamChatClient {
   ///```
   final LogHandlerFunction logHandlerFunction;
 
-  StreamSubscription<ConnectionStatus>? _connectionStatusSubscription;
+  StreamSubscription<List<ConnectionStatus>>? _connectionStatusSubscription;
```

Comment:
Isn't it better to make this a (named) record instead of a list? Should also be 2 states right?

Reply:
Ah right, I thought the `pairwise()` was something from us, but it's a feature from rxdart.